bigbluebutton-html5/imports/api/users/server/publishers.js

```diff
@@ -57,11 +57,14 @@ function users(credentials, isModerator = false) {
   };
 
   if (isModerator) {
-    selector.$or.push({
-      'breakoutProps.isBreakoutUser': true,
-      'breakoutProps.parentId': meetingId,
-      connectionStatus: 'online',
-    });
+    const User = Users.findOne({ userId: requesterUserId });
+    if (User.moderator) {
+      selector.$or.push({
+        'breakoutProps.isBreakoutUser': true,
+        'breakoutProps.parentId': meetingId,
```

Comment:
Should not these be in an $and condition, not an $or condition?

Reply:
Nope, I wouldn't be able to see any, the first is to see the meeting users the second is to see user that joined in breakout and allow invite user that didn' join yet